src/main/java/io/confluent/examples/streams/microservices/FraudService.java

```diff
@@ -75,6 +75,7 @@ public void start(final String bootstrapServers, final String stateDir) {
     log.info("Started Service " + getClass().getSimpleName());
   }
 
+  @SuppressWarnings("unchecked")
```

Comment:
Suppress warning about untypes `branch()` operator.

Reply:
Can you elaborate? This sounds like we designed a flaw into the API, such that anyone using `branch` would have to suppress this compiler warning.

Reply:
Could we do https://docs.oracle.com/javase/7/docs/api/java/lang/SafeVarargs.html on the `branch` interface, since this would be required by every caller? If not, it would still be cleaner to move this line to the call site to minimize the scope of warning suppression.

Reply:
Sounds good. I would prefer to fix this properly too. In the meantime, I would suggest moving the annotation down to the statement that has the problem versus doing it at method scope.

Reply:
A statement-level suppression may satisfy IDEA but not the compiler, the only way to know is to try.

Regardless, I'm fine with adding the suppression. Thanks for the explanation.

Reply:
`mvn clean compile test-compile` did not give a warning any longer, even with the statement-level suppression -- the compiler seems to be happy :)